### PR TITLE
[type: performance] Implement zero-copy in TCP plugin

### PR DESCRIPTION
**Pull request by TeslaCN**
Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

## Before

a4aa2e16-7a89-4634-99fb-44534d82024e

## After

f50edb51-2785-4c69-a5ac-b37ab9ade4d6

### CODE REVIEW

**Review by GOODBOY008 — APPROVED** [review]
@TeslaCN  LGTM, Can you remove useless import `import reactor.core.publisher.Flux;
import reactor.core.publisher.Mono;`

File: shenyu-protocol/shenyu-protocol-tcp/src/main/java/org/apache/shenyu/protocol/tcp/connection/TcpConnectionBridge.java [hunk 1]
```diff
@@ -46,8 +46,7 @@ public void bridge(final Connection server, final Connection client) {
     }
 
     private Disposable bridge(final NettyInbound inbound, final NettyOutbound outbound) {
-        Flux<byte[]> flux = inbound.receive().asByteArray();
-        return flux.concatMap(next -> outbound.sendByteArray(Mono.just(next))).subscribe();
+        return outbound.send(inbound.receive().retain()).then().subscribe();
```
Comment (GOODBOY008) [inline shenyu-protocol/shenyu-protocol-tcp/src/main/java/org/apache/shenyu/protocol/tcp/connection/TcpConnectionBridge.java:49]: Remove useless import `import reactor.core.publisher.Flux; import reactor.core.publisher.Mono;`.

Comment (TeslaCN) [reply shenyu-protocol/shenyu-protocol-tcp/src/main/java/org/apache/shenyu/protocol/tcp/connection/TcpConnectionBridge.java:47]: Done.

**Review by yu199195 — APPROVED** [review]

**Review by moremind — APPROVED** [review]

**Comment by ywj1352** [comment]
NB

**Pull request merged**